pytorch_lightning/loops/batch/training_batch_loop.py

```diff
@@ -674,3 +674,9 @@ def _truncated_bptt_steps(self) -> int:
         if lightning_module.truncated_bptt_steps > 0:
             return lightning_module.truncated_bptt_steps
         return self.trainer.truncated_bptt_steps or 0
+
+    def state_dict(self) -> Dict:
+        return {}
```

Comment:
The base class could return the empty dict itself

Reply:
Done !